src/types/formatted/benefitsHub.ts

```diff
@@ -1,13 +1,10 @@
 export type BenefitsHubLink = {
   id: string
-  title: string
```

Comment:
Removed these extra props as they aren't actually coming in from the benefits hub link data